index.d.ts

```diff
@@ -60,3 +60,26 @@ console.log(await getPort({port: portNumbers(3000, 3100)}));
 ```
 */
 export function portNumbers(from: number, to: number): Iterable<number>;
+
+/**
+Clear the internal cache of locked ports.
+
+When you want your results to be unaffected by previous calls.
```

Comment:
That is implied by the function named. It should explain the actual use-cases for the function.

Reply:
Sorry, I didn't quite understand what the specific actual scenario refers to. For me, I just want the result of each call
Not affected by previous calls.